src/Test/Docs.hs

```diff
@@ -21,15 +21,14 @@ main = shaken noTest $ \args obj -> do
 
     config %> \_ -> do
         need ["shake.cabal"]
-        unit $ cmd "runhaskell Setup.hs configure" ["--builddir=" ++ obj "dist","--user"]
+        unit $ cmd "cabal configure" ["--builddir=" ++ obj "dist","--user"]
```

Comment:
I run the suite inside the sandbox as:

    env FSAT=<path-to-fsatrace> cabal exec -- cabal test

I can try again, but I think runhaskell isn't affected by CABAL_SANDBOX_PACKAGE_PATH.